BUILD

```diff
@@ -8,3 +8,18 @@ shell_sources(name="scripts", sources=["cargo", "pants"])
 files(name="files", sources=["BUILD_ROOT", "pants.toml"])
 
 python_test_utils(name="test_utils")
+
+_local_environment(
+    name="macos_local_env",
+    # Avoid system Python interpreters, which tend to be broken on macOS.
+    python_interpreter_search_paths=["<PYENV>"],
```

Comment:
Note that is functionality we could not have before! We should confirm we actually want to land this for pantsbuild/pants developers. Thoughts?

Reply:
Works for me... I set this via `~/.pants.rc